### PR TITLE
added werkk freelancing platform

### PR DESCRIPTION
**Pull request by ASACHIT**
Added a New Company: [werkk freelancing platform](http://werkk.tech/)

### CODE REVIEW

**Comment by TheLearneer** [comment]
@ASACHIT is Werkk an officially registered company?

**Comment by ASACHIT** [comment]
> @ASACHIT is Werkk an officially registered company in Nepal?

yes it is registered

File: data/werkk.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "werkk freelancing platform",
+  "establishedAt": 1663987020,
+  "location": "Fully remote",
+  "website": "http://werkk.tech/",
+  "email": "werkk@noditt.tech",
+  "phone": "+977-9828104810",
+  "socials": {
+    "facebook": "https://www.facebook.com/werkk.tech",
+    "facebook_group": "https://www.facebook.com/groups/werkk",
```
Comment (TheLearneer) [inline data/werkk.json:10]: Sorry but please remove `facebook_group` key.
Also even if the other social keys have null value, still include those 🙏

**Review by TheLearneer — CHANGES_REQUESTED** [review]

File: data/werkk.json (hunk 1 above)
Comment (ASACHIT) [reply data/werkk.json:10]: sure !

**Review by TheLearneer — APPROVED** [review]

**Pull request merged**